src/dbnode/persist/fs/read.go

```diff
@@ -205,12 +205,12 @@ func (r *reader) Open(opts DataReaderOpenOptions) error {
 	result, err := mmap.Files(os.Open, map[string]mmap.FileDesc{
 		indexFilepath: mmap.FileDesc{
 			File:    &r.indexFd,
-			Bytes:   &r.indexMmap,
+			Bytes:   &r.indexMmap.Bytes,
```

Comment:
Looks like we miss setting `ReporterOptions` in `mmap.Options` below for both these mmaps?